src/cmd/prune.ts

```diff
@@ -0,0 +1,9 @@
+import prune from '../api/prune'
+import {PublicInstallationOptions} from '../api/install'
+
+export default (input: string[], opts: PublicInstallationOptions) => {
+  if (input && input.length) {
+    return prune(input, opts)
+  }
+  return prune(opts)
```

Comment:
Just a note, for the install there are two separate functions: `pnpm.install(pkgsToInstall, [options])` and `pnpm.installPkgDeps([options])`

So we have to decide between changing the order of arguments or creating a dedicated function.